src/pages/Home/StoreLocator.js

```diff
@@ -23,83 +34,69 @@ const StoreList = styled.div`
   flex-wrap: wrap;
   align-items: center;
   justify-content: space-evenly;
-  width: 90%;
+  width: 100%;
   max-width: 450px;
-  margin: 0 auto;
+  margin: auto;
+
+  @media ${device.mobile} {
+    text-align: center;
+    align-content: center;
+    margin: auto;
+  }
   @media ${device.tablet} {
     flex-direction: column;
-    max-width: '';
-    margin: 0 auto 0 20px;
-    width: 60%;
+    text-align: center;
```

Comment:
we should be able to remove this text-align since it's already declared the same at the mobile bp

Reply:
@dbridenbeck removed